### DISCUSSION TITLE
Meeting Agenda 4/2

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#361] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Sprint Planning
3. Finalize expo poster
    - Make sure that all visual items are big enough
4. Discuss/finalize pitch
4. Meet with Darian

**Comment by moreno-michael** [comment]
Meeting minutes 4/2

1. Task Review
- All tasks still in progress
- Legend and load balancer configurability are pretty much the last coding intensive tasks

2. Sprint planning
- Tasks from previous sprint pushed to this sprint

3. Expo discussion
- Node refresh animation changed to better accommodate photosensitive users
- Resize and edit poster to better make use of poster board we will be given (more diagrams, multiple pages of a poster)

4. Discuss pitch
- Carson and Chris will record competitive pitch tomorrow (time tbd, due @ midnight and must be uploaded to yt)
- Mike and Victor will do non-competitive pitch (due 4/10)

5. Meet with Darian
- Task Review
- Poster overall looks good
- Maybe run poster by an artist/graphic designer to be more marketable
- Figure out how to implement chatbot?
